src/toil/test/__init__.py

```diff
@@ -253,3 +253,139 @@ def needs_cwl(test_item):
         raise
     else:
         return test_item
+
+MethodNameRegex = re.compile('[a-zA-Z_0-9]+$')
```

Comment:
this regex checks if a string could be part of a method name, and every method name used int he make_tests function is prepended with "test_" which is why digits are included